### PR TITLE
Wraps p5.dom to support inclusion via amd and browserify

### PR DESCRIPTION
**Pull request by brysonian**
amd style:
```require(["p5", "p5.dom"], function(p5) {...});```

browserify style:
```
var p5 = require("p5");
require("p5/lib/addons/p5.dom");
```

Also preserves support for "normal" use via script tags.

### CODE REVIEW

File: lib/addons/p5.dom.js [hunk 1]
```diff
@@ -211,23 +217,21 @@ var p5DOM = (function(){
...
    * @return {Object/p5.Element} pointer to p5.Element holding created
    *                           node
    */
-  p5.prototype.createSlider = function(min, max, value, step) {
+  p5.prototype.createSlider = function(min, max, value) {
     var elt = document.createElement('input');
     elt.type = 'range';
     elt.min = min;
     elt.max = max;
-    if (step) elt.step = step;
```
Comment (kadamwhite) [inline lib/addons/p5.dom.js]: This change doesn't immediately appear to be AMD-related, was this intended?

Comment (brysonian) [inline lib/addons/p5.dom.js]: hrm no, it looks like it reverts https://github.com/lmccart/p5.js/pull/504/files damn it. i’ll do another pr to restore.





--------------------------------------------------Chandler McWilliamshttp://brysonian.com

**Comment by lmccart** [comment]
great catch @kadamwhite thanks for taking a look!
